File: Samples/QueueBrowser/app/models/LogError.ts
/// <reference path="../_ref.ts" />
module Messaging {
  "use strict";

  export interface ILogError {
    Message?: string;
    BaseMessage?: string;
    Text?: string;
    Type?: string;
    Source?: string;
    MethodName?: string;
    ModuleName?: string;
    ModuleVersion?: string;
    ErrorCode?: number;
  }
}

File: Samples/QueueBrowser/Models/LogError.cs
using System;
using MongoDB.Bson.Serialization.Attributes;

namespace QueueBrowser.Models
{
    [BsonIgnoreExtraElements(true, Inherited = true)]
    public class LogError
    {
        public string Message { get; set; }

        public string BaseMessage { get; set; }

        public string Text { get; set; }

        public string Type { get; set; }

        public string Source { get; set; }

        public string MethodName { get; set; }

        public string ModuleName { get; set; }

        public string ModuleVersion { get; set; }

        public int? ErrorCode { get; set; }
    }
}